### PR TITLE
Bruce 6120:UT coverage for svc-aggregation

### PR DESCRIPTION
**Pull request by lpooja**

### CODE REVIEW

**Review by jeevan-kamkar — APPROVED** [review]

File: svc-aggregation/system/pluginHealthCheck_test.go [hunk 1]
```diff
@@ -0,0 +1,233 @@
+/* (C) Copyright [2020] Hewlett Packard Enterprise Development LP
```
Comment (chauberahul1993) [inline svc-aggregation/system/pluginHealthCheck_test.go]: Update license year

File: svc-aggregation/system/pluginHealthCheck_test.go [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+// Package system ...
+
+package system
+
+import (
+	// "encoding/json"
+	//"io/ioutil"
+	//"net/http"
+	//"reflect"
```
Comment (chauberahul1993) [inline svc-aggregation/system/pluginHealthCheck_test.go]: remove comments

File: svc-aggregation/system/pluginHealthCheck_test.go [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+		}
+	}()
+	startUpData := agmodel.PluginStartUpData{
+		RequestType:           "full",
+		ResyncEvtSubscription: true,
+	}
+	var startUpData1 interface{}
+	sendPluginStartupRequest(agmodel.Plugin{}, startUpData1, "")
+
+	sendPluginStartupRequest(agmodel.Plugin{}, startUpData1, "ILO_v1.0.0")
```
Comment (chauberahul1993) [inline svc-aggregation/system/pluginHealthCheck_test.go]: Add assert checks

File: svc-aggregation/system/pluginHealthCheck_test.go [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+	plugin := agmodel.Plugin{
+		IP:                "localhost",
+		Port:              "1234",
+		Password:          password,
+		Username:          "admin",
+		ID:                "GRF",
+		PreferredAuthType: "BasicAuth",
+		PluginType:        "GRF",
+	}
+	//plugin1 := agmodel.Plugin{
```
Comment (chauberahul1993) [inline svc-aggregation/system/pluginHealthCheck_test.go]: remove comments

**Review by chauberahul1993 — APPROVED** [review]

File: svc-aggregation/system/pluginHealthCheck_test.go (hunk 1 above)
Comment (lpooja) [reply svc-aggregation/system/pluginHealthCheck_test.go]: updated

File: svc-aggregation/system/pluginHealthCheck_test.go (hunk 4 above)
Comment (lpooja) [reply svc-aggregation/system/pluginHealthCheck_test.go]: updated

**Review by hemkapoor — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**